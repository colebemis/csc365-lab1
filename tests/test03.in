G: 1 h
Q