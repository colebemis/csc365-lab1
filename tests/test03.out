Enter query: Querying highest GPA in grade 1
GARTH, JOHN, 3.14, FALKER, ADOLPH, 0
Enter query: 